wurst/dummy/InstantDummyCaster.wurst

```diff
@@ -16,51 +16,55 @@ public class InstantDummyCaster
 	
 	private construct()
 	
-	private static function prepare(player owner, integer lvl)
+	private static function prepare(player owner, integer abilityId, integer lvl, vec2 pos)
+		id = abilityId
 		caster.addAbility(id)
+		caster.setXY(pos)
 		if lvl > 1
 			caster.setAbilityLevel(id, lvl)
 		if owner != null
 			caster.setOwner(owner, false)
 			
 	private static function finish()
-		caster..setOwner(DUMMY_PLAYER, false)..removeAbility(id)
```

Comment:
Why did you convert ".." to "caster." ????
.. is the recommended convention, please restore it.